test/travis_ci.sh

```diff
@@ -142,6 +157,15 @@ function init_confd {
     printf "\nInitializing confd\n"
 }
 
+function init_confd_model_test {
+    confd --stop
+    cp -f $YDKTEST_MODEL_FXS/* $CONFD_TARGET_DIR
```

Comment:
I am thinking about split `CONFD_TARGET_DIR` for each different test cases. The `ydktest`, `ydk-model-test`, `deviation`, and `augmentation test` I am going to add will have their `CONFD_<test-case-name>_TARGET_DIR`, so the fxs files for different test cases will not mess up. Each `CONFD_<test-base-name>_TARGET_DIR` has same content except for their configuration file `confd.conf`(its relatively easy to modify confd.conf). Let me know what you think.